source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
@aabrook the component is good, but it's still only rendering on the client side.
Which is fine, but if you `renderToStaticMarkup(<App />)` in `renderDocument` on the server side, it'd be built into the resulting `index.html` file